### DISCUSSION TITLE
30 January–3 February 2023

### DISCUSSION THREAD

**Discussion by esheehan-gsl** [NOAA-GSL/unified-graphics#146] [Progress Report]
🚴🏻 **Progress**

- #135 runs locally and is ready for testing in AWS

😖 **Regressions**

None

💬 **To Discuss**

None

📆 **Next Week**

- Updating the backend to read from Zarr arrays instead of NetCDF files

**Comment by ian-noaa** [comment]
🚴🏻 **Progress**

- Worked with VLab to set up a container registry for the data pipeline
- Created cost estimates for the next year
- #117 

😖 **Regressions**

None

💬 **To Discuss**

None

📆 **Next Week**

- Creating a S3 bucket for production data
- Testing the data pipeline in VLab
- Discuss reserved instances with VLab
